Scripts/travis/test-Linux.sh

```diff
@@ -0,0 +1,17 @@
+# Run integration tests on Linux
+set -eu
+
+if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  TARGET_BRANCH="$TRAVIS_BRANCH"
+  GIT_SLUG="$TRAVIS_REPO_SLUG"
+else
+  TARGET_BRANCH="$TRAVIS_PULL_REQUEST_BRANCH"
+  GIT_SLUG="$TRAVIS_PULL_REQUEST_SLUG"
+fi
+
+GIT_PATH="https://github.com/$GIT_SLUG.git"
+
+# we want to use the dockerfile
+docker build - < Dockerfile -t restkit:linux-tests --build-arg git_path=$GIT_PATH --build-arg git_branch=$TARGET_BRANCH
+docker run restkit:linux-tests
+docker system prune -a -f
```

Comment:
If this script is for running the tests in a Travis Linux environment, then there is no need to spin up a docker container.  We already have Linux, so we should just ensure that the Swift compiler is installed and then run the tests directly.

I'd like this script to look very much like the `test-macOS.sh` travis script, where it simply calls the appropriate scripts in the `Scripts` directory.

Reply:
We can install swift in the existing Linux image that Travis is running in, but some of the advantages of using the swift image and running it in a container are additional packages that the swift image installs, as well as a GPG signing on the packages.

I get the feeling that running a container in a container can feel unnecessary, but to me it feels more stable than `wget`ing a swift toolchain from a static url

Reply:
Right ... running a container in a container feels unnecessary, but I see your point that it might be the most straightforward way to create the environment needed.